design/features/file-upload/README.md

```diff
@@ -0,0 +1,353 @@
+
+# File upload first release
+
+## Status 
+
+Date created: *2025-07-09*  
+
+* * *
+
+## Contents
+
+- [What](#what)
+    - [Our users](#our-users)
+    - [Previous research](#previous-research)
+    - [The problem](#the-problem)
+    - [User stories](#user-stories)
+- [Key decisions](#key-decisions)
+- [Feedback from users](#feedback-from-users)
+- [Released feature](#released-feature)
+    - [Form creator can add a file upload question](#form-creator-can-add-a-file-upload-question)
+    - [Form creator edits file upload question](#form-creator-edits-file-upload-question)
+    - [Upload a file - form filler user flow](#upload-a-file---form-filler-user-flow)
+    - [Upload a file - form filler](#upload-a-file---form-filler)
+    - [Check your uploaded file - form filler](#check-your-uploaded-file---form-filler)
+    - [Confirm to remove your file - form filler](#confirm-to-remove-your-file---form-filler)
+    - [File removed, upload a new one - form filler](#file-removed-upload-a-new-one---form-filler)
+    - [Check your answers with file upload](#check-your-answers-with-file-upload)
+    - [Check your answer page with page heading for file upload question with guidance](#check-your-answer-page-with-page-heading-for-file-upload-question-with-guidance)
+    - [Submitted form responses in an email to a form processor](#submitted-form-responses-in-an-email-to-a-form-processor)
+- [Measuring](#measuring)
+- [What we didn’t do and why](#what-we-didnt-do-and-why)
+- [What would we like to do in the next release](#what-would-we-like-to-do-in-the-next-release)
+
+* * *
+
+## What
+
+Epic trello card: [https://trello.com/c/NWsgTJrc](https://trello.com/c/NWsgTJrc)
+
+[Mural working board](https://app.mural.co/t/gaap0347/m/gaap0347/1731576709302/ed0fdc4d589481830738715cff7b00d6b4f268e3?sender=u1786fa2d2029c0638a5c0139)
+
+### Our users
+
+**Form creator** is a person who builds a digital form using GOV.UK Forms
+
+**Form filler** is a person who fills in the form and submits it in order to receive a service
+
+**Form processor** is a person who receives the answers from the form submission and uses the information to provide a service
+
+### Previous research
+
+ [File upload and optional questions research playback December 2022](https://docs.google.com/presentation/d/1fFSjoXwM0rO0sJHDHRUak9a4aphiutl8u7HYlYdqX1c/edit#slide=id.g1ab61843043_0_71) 
+
+[File upload research insights November 2022](https://docs.google.com/document/d/1MDc2X3MSZJhkKLyyRb8dWapPthQAV_sMPaBC8q82zkQ/edit?usp=sharing) 
+
+People currently asking for lists of documents to be sent in could see the benefit of structuring this request with a question for each. At the moment their form completers often miss some documents leading to correspondence back and forth. One estimated 90% of submissions are currently incomplete.
```

Comment:
```suggestion
Form creators who currently ask for lists of documents to be sent in could see the benefit of structuring this request with a question for each document. At the moment their form fillers often miss some documents leading to correspondence back and forth. One estimated 90% of submissions are currently incomplete.
```

Worth making it clear who the people are, in this case ‘form creators’ and aligning the ‘form filler’ language.